src/main/java/org/webonise/listapp/UserComparator.java

```diff
@@ -0,0 +1,23 @@
+package org.webonise.listapp;
+
+import java.util.Comparator;
+
+public class UserComparator implements Comparator<Users> {
+    public int compare(Users user1, Users user2) {
+        int result = 0;
+        result = user1.getId() - user2.getId();
+        if(result == 0){
+            result = user1.getFirstName().compareTo(user2.getFirstName());
```

Comment:
Can be simplified as if ( user1.getFirstName().compareTo(user2.getFirstName()) == 1 ) {
result = 1;
}  else if ( ... ) {
}